src/main/java/org/example/Main.java

```diff
@@ -1,7 +1,120 @@
 package org.example;
 
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.*;
+
 public class Main {
+
+    static List<String> options = new ArrayList<>(List.of("add", "remove", "list", "exit"));
+    static List<List<String>> tasks = new ArrayList<>();
+    static Scanner scanner = new Scanner(System.in);
+
     public static void main(String[] args) {
-        System.out.println("Hello, World!");
+        readTasks("tasks.csv");
```

Comment:
CLEAN-CODE: lepsi popis by byl loadTasks() a parametr bych da do globalni promenne v ramci tridy nahoru neco ve stylu TASKS_FILE s kterym bych pak pracoval uvnitr metody loadTasks().
```suggestion
        loadTasks();
```